src/test/resources/SubscriptionObjectForEmailTest.json

```diff
@@ -0,0 +1,32 @@
+{
+        "subscriptionName": "Subscription_1",
+        "repeat": false,
+        "created": "data-time",
+        "notificationType": "MAIL",
+        "notificationMeta": "mohammad.saifullah@ericsson.com, msdjah, saif.phy@yahoo.com",
```

Comment:
Remove your own mail code. Its ok to use it for testing locally, but remember to replace with some other non existing mail address.